DigitalExcellence/dex-frontend
# Feature/451-add-call-to-action-icons

## Description

This pull request will automatically add the icons from the design to the call to action button, I also added a hotfix for the wizard where the project could not be uploaded if you didn't add a project icon.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project with a call-to-action
2. Go to the detail page of the created project
3. The call-to-action-button should have an icon on the right

## Link to issue

Closes: #451
